sympy/physics/hep/gamma_matrices.py

```diff
@@ -25,7 +25,8 @@
     metric(LorentzIndex,LorentzIndex)
 
 """
-from sympy import S, Mul, eye, trace
+from sympy.core.backend import Mul, eye
+from sympy import trace, S
```

Comment:
I meant about that only. The only way that occurs to me is to implement the `Singleton` pattern in `SymEngine`. Is there a workaround for this?